### PR TITLE
APP-15991 chip adjustments

### PR DESCRIPTION
**Pull request by Sqrrl**
## Summary

- [Ticket](https://flipapp.myjetbrains.com/youtrack/issue/APP-15991/Swirl-Chip-adjustments)
- [Design](https://www.figma.com/file/iRHiAFsHxnzux8gQ9EjlZe/Swirl-Components?node-id=2%3A19&mode=dev)
- [Storybook](https://icy-water-049ec4003-338.westeurope.3.azurestaticapps.net/?path=/docs/components-swirlchip--docs)

### General

- [ ] [Changeset added](https://github.com/changesets/changesets/blob/main/docs/intro-to-using-changesets.md)
- [ ] The submitted code is organized and formatted according to our [💅 Style Guide](https://swirl-storybook.flip-app.dev/?path=/docs/requirements-style-guide--docs).
  - [ ] The code is formatted with Prettier.
  - [ ] There are no linting errors.

### For new or updated components

https://swirl-storybook.flip-app.dev/?path=/docs/contributions-merge-publish--page

- [ ] The changes do not contain any breaking changes.
- [ ] The changes do not introduce new components that don't belong in the library (e.g. non-reusable components, highly specialized components, components including business logic)
- [ ] The component documentation is updated.
- [ ] The changes meet the [🤖 testing requirements](https://swirl-storybook.flip-app.dev/?path=/docs/requirements-testing--docs).
  - [ ] New features are tested.
  - [ ] In case of bug fixes, regression tests have been added.
  - [ ] All tests are 🟢.
- [ ] The changes meet the [🧏‍♀️ accessibility requirements](https://swirl-storybook.flip-app.dev/?path=/docs/requirements-accessibility--docs).
  - [ ] WCAG 2.1 Level A and Level AA requirements are met.
  - [ ] The Storybook a11y addon shows no errors.
  - [ ] The changes have been tested with a screen reader.
  - [ ] Keyboard controls have been tested, if applicable.
  - [ ] Components implementing form controls (inputs, buttons, selects, etc.) do not use Shadow DOM, and instead use Stencil's scoping mechanism
- [ ] The changes use our [🌈 theming concept](https://swirl-storybook.flip-app.dev/?path=/docs/requirements-theming--docs).
  - [ ] Design tokens have been used where appropriate.
  - [ ] The component has been visually checked in combination with the "Light" and "Dark" theme.
- [ ] The changes meet our [🌍 internationalization requirements](https://swirl-storybook.flip-app.dev/?path=/docs/requirements-internationalization--docs).
  - [ ] No static text is used.
  - [ ] The component doesn't break with longer texts or different text wrappings.
  - [ ] Number, currency and date values can be formatted appropriately.
- [ ] The changes work in all supported browsers and viewports. See [📱 Responsive Design](https://swirl-storybook.flip-app.dev/?path=/docs/requirements-responsive-design--docs)

### CODE REVIEW

File: packages/swirl-components/src/components/swirl-chip/swirl-chip.css [hunk 1]
```diff
@@ -54,11 +75,23 @@
...
+
+    & + .chip__remove-button {
+      background-color: var(--s-surface-overlay-pressed);
+    }
+  }
+}
+
+.chip--icon-color-highlight {
+  & .chip__icon {
+    color: var(--s-icon-highlight);
```
Comment (webspaceadam) [inline packages/swirl-components/src/components/swirl-chip/swirl-chip.css:87]: **issue (blocking):** seems that this style does not work. i also don't see this class correclty rendered in the browser styles
image

**Review by webspaceadam — CHANGES_REQUESTED** [review]
Seems there is a small issue regarding the icon color

File: packages/swirl-components/src/components/swirl-chip/swirl-chip.css (hunk 1 above)
Comment (Sqrrl) [reply packages/swirl-components/src/components/swirl-chip/swirl-chip.css:87]: The icon color only affects the icon passed via the icon prop.

Comment (webspaceadam) [reply packages/swirl-components/src/components/swirl-chip/swirl-chip.css:87]: ah ok. my bad then :D 
works!

**Review by webspaceadam — APPROVED** [review]

**Pull request merged**